ci/prow/config.yaml

```diff
@@ -1582,6 +1602,47 @@ periodics:
       resources:
         requests:
           memory: "1Gi"
+- cron: "1 9 * * 2" # Run at 02:01PST every Tuesday (09:01 UTC)
+  name: ci-knative-serving-dot-release
+  agent: kubernetes
+  labels:
+    preset-release-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
```

Comment:
These Bazel settings seem weird.

Reply:
As I said, I haven't bothered fine tunning settings again for the past 6 months, probably it's a good time to do it again. Filed #326.